example-crates/example_cli/src/lib.rs

```diff
@@ -667,15 +667,15 @@ pub fn handle_commands<CS: clap::Subcommand, S: clap::Args>(
                     // print base64 encoded psbt
                     let fee = psbt.fee()?.to_sat();
                     let mut obj = serde_json::Map::new();
-                    obj.insert("psbt".to_string(), json!(psbt.to_string()));
+                    obj.insert("psbt".to_string(), json!(psbt));
```

Comment:
I think `to_string` is needed to get the PSBT in base64. You can debug print the struct using `-d`

Reply:
Good catch! fixed.